nesta_ds_utils/file_ops.py

```diff
@@ -1,53 +1,431 @@
+    elif fnmatch(path_to, "*.parquet"):
+        df_data.to_parquet(buffer, **kwargs)
+    else:
+        raise Exception(
+            "Uploading dataframe currently supported only for 'csv' and 'parquet'."
+        )
+    buffer.seek(0)
+    return buffer
+
+
+def _dict_to_fileobj(dict_data: dict, path_to: str, **kwargs) -> io.BytesIO:
+    """Convert dictionary into bytes file object.
+
+    Args:
+        dict_data (dict): Dictionary to convert.
+        path_to (str): Saving file name.
+
+    Returns:
+        io.BytesIO: Bytes file object.
+    """
+    buffer = io.BytesIO()
+    if fnmatch(path_to, "*.json"):
+        buffer.write(json.dumps(dict_data, **kwargs).encode())
+    else:
+        raise Exception("Uploading dictionary currently supported only for 'json'.")
+    buffer.seek(0)
+    return buffer
+
+
+def _list_to_fileobj(list_data: list, path_to: str, **kwargs) -> io.BytesIO:
+    """Convert list into bytes file object.
+
+    Args:
+        list_data (list): List to convert.
+        path_to (str): Saving file name.
+
+    Returns:
+        io.BytesIO: Bytes file object.
+    """
+    buffer = io.BytesIO()
+    if fnmatch(path_to, "*.csv"):
+        pd.DataFrame(list_data).to_csv(buffer, **kwargs)
+    elif fnmatch(path_to, "*.txt"):
+        for row in list_data:
+            buffer.write(bytes(str(row) + "\n", "utf-8"))
+    elif fnmatch(path_to, "*.json"):
+        buffer.write(json.dumps(list_data, **kwargs).encode())
+    else:
+        raise Exception(
+            "Uploading list currently supported only for 'csv', 'txt' and 'json'."
+        )
+    buffer.seek(0)
+    return buffer
+
+
+def _str_to_fileobj(str_data: str, path_to: str, **kwargs) -> io.BytesIO:
+    """Convert str into bytes file object.
+
+    Args:
+        str_data (str): String to convert.
+        path_to (str): Saving file name.
+
+    Returns:
+        io.BytesIO: Bytes file object.
+    """
+    if fnmatch(path_to, "*.txt"):
+        buffer = io.BytesIO(bytes(str_data.encode("utf-8")))
+    else:
+        raise Exception("Uploading string currently supported only for 'txt'.")
+    buffer.seek(0)
+    return buffer
+
+
+def _np_array_to_fileobj(
+    np_array_data: np.ndarray, path_to: str, **kwargs
+) -> io.BytesIO:
+    """Convert numpy array into bytes file object.
+
+    Args:
+        np_array_data (np.ndarray): Numpy array to convert.
+        path_to (str): Saving file name.
+
+    Returns:
+        io.BytesIO: Bytes file object.
+    """
+    buffer = io.BytesIO()
+    if fnmatch(path_to, "*.csv"):
+        np.savetxt(buffer, np_array_data, delimiter=",", **kwargs)
+    elif fnmatch(path_to, "*.parquet"):
+        pq.write_table(pa.table({"data": np_array_data}), buffer, **kwargs)
+    else:
+        raise Exception(
+            "Uploading numpy array currently supported only for 'csv' and 'parquet."
+        )
+    buffer.seek(0)
+    return buffer
+
+
+def _unsupp_data_to_fileobj(data: any, path_to: str, **kwargs) -> io.BytesIO:
```

Comment:
```suggestion
def _unsupp_data_to_fileobj(data: Any, path_to: str, **kwargs) -> io.BytesIO:
```

This is using the Python built-in `any` function as a type hint, rather than `typing.Any` which is the generic object type.